src/Runtime/LambdaRuntime.php

```diff
@@ -94,6 +94,10 @@ public function processNextEvent($handler): bool
         } catch (\Throwable $e) {
             $this->signalFailure($context->getAwsRequestId(), $e);
 
+            if (in_array(getenv('BREF_INVOKE_LAMBDA_ERROR_THROW_EXCEPTIONS'), ['true', '1'], true)) {
```

Comment:
`getenv` is not really suitable for this. `$_SERVER` is always the best place for env variable reading and writing... yes PHP did a super bad job of naming things.

Reply:
I have no argument againts that - can you elaborate on that?

Also, if i grep in the package for `getenv` i'ts all over the place.

Reply:
We indeed use getenv all over the place, it's single-process single-thread everywhere, any issue we are risking?